python/lsst/pipe/tasks/imageDifference.py

```diff
@@ -348,6 +401,39 @@ def makeIdFactory(expId, expBits):
         """
         return afwTable.IdFactory.makeSource(expId, 64 - expBits)
 
+    def runQuantum(self, butlerQC: pipeBase.ButlerQuantumContext,
+                   inputRefs: pipeBase.InputQuantizedConnection,
+                   outputRefs: pipeBase.OutputQuantizedConnection):
+        """Pipelinetask gen3 entry point.
+
+        Parameters
+        ----------
+        expId : `int`
+            Exposure id.
+
+        expBits: `int`
+            Number of used bits in ``expId``.
+
+        Returns
+        -------
+        None
+        """
```

Comment:
This docstring looks copy-pasted from above. I think you actually want to delete it, and add a `@utils.inheritDoc(pipeBase.PipelineTask)` decorator above since these are just the standard parameters for a pipelineTask.